yaks/lib/yaks/reader/hal.rb

```diff
@@ -29,7 +29,7 @@ def convert_links(links)
             rel = rel.to_sym if rel =~ /\A\w+\z/
             Resource::Link.new(rel: rel, uri: l['href'], options: options)
           end
-        end.to_set
+        end
```

Comment:
Ok thanks for the explanation. So it has nothing to do with uniqueness of items in set?